mobify/progressive-web-scaffold
# Get bundle live on site

One option to get the scaffold bundle live on https://www.merlinspotions.com - just push to the existing Mobify Cloud project. 

 **JIRA**: https://mobify.atlassian.net/browse/WEB-992

## Changes
- Set the `package.json` `name` to `platform-scaffold`
- Add a `projectSlug` value to `package.json` - use that for Cloud project name (uploading bundles) and `a.js` loading parameter

## How to test-drive this PR
- Run Preview (see `npm run dev` output)
- Observe the Web Push ask still occurs (make sure to emulate a non-iOS browser, clear cookies/site data as required to get the ask to show)
